images_to_samples.py

```diff
@@ -326,8 +328,7 @@ def main(params):
         samples_folder = data_path.joinpath(sample_path_name)
 
     if samples_folder.is_dir():
-        warnings.warn(f'Data path exists: {samples_folder}. Suffix will be added to directory name.')
-        samples_folder = Path(str(samples_folder) + '_' + now)
+        raise FileExistsError(f'Data path exists: {samples_folder}. Remove it or use a different run_name.')
```

Comment:
I agree to change this behavior. During training, I think it's ok to automatically add a suffix if the directory already exists. During sample preparation, the opposite behavior (raise exception) makes sense. An alternative approach would be to keep old behavior only if in "debug" mode. I found it useful to have an automatic directory creation when I was debugging in the past...

Reply:
second though, for debugging, it's really fun to not raise an exception if directory exists (i.e. when having to run images_to_samples multiple times in a small time lapse)

Reply:
my confusion is that train_segmentation always uses the first one (without the added datetime), so it feels like rerunning images_to_samples is a dead end, because if you like the new results, they are not in a directory used by train_segmentation. Perhaps debug mode should instead move the old directory (with a random suffix?). To me it seems like a mistake waiting to happen to rerun images_to_samples, and then run train_segmentation thinking the new samples will be used.

Reply:
your absolutely right. The approach we had is not robust. moving the old directory seems like a good fix, again, just for debugging purposes.

Reply:
New proposition now moves the old directory...